recipes/novas/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "novas" %}
+{% set version = "3.1.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/42/95/a05bc35cb119925e10f9faa8a2bd17020b0a585744a38921a709acdd9a14/{{ name }}-{{ version }}.tar.gz
```

Comment:
This should be the predictable URL (see `recipes/example/meta.yaml` for how to construct it)

Reply:
Done, copied from `recipes/example/meta.yml`